network/junos/junos_command.py

```diff
@@ -183,9 +183,9 @@ def parse(module, command_type):
             item = dict(command=item, output=None)
         elif 'command' not in item:
             module.fail_json(msg='command keyword argument is required')
-        elif item.get('output') not in [None, 'text', 'xml']:
+        elif item.get('output') not in [None, 'text', 'xml', 'json']:
```

Comment:
@gundalow  I see @privateip  added this in PR #4687. Unsure what this would add and might be worth adding to https://github.com/ansible/community/issues/110.